:w

```diff
@@ -0,0 +1,8 @@
+const { JSDOM } = require('jsdom');
```

Comment:
I'm curious about the name of this file as :w. Is that needed or could it be something else? It is unusual. It seems similar to src/server/services/utils/sanitizer.js so I wonder if it is needed.